Resources/Private/Fusion/Prototypes/ActionCollection.Fieldnames.fusion

```diff
@@ -0,0 +1,11 @@
+prototype(Sitegeist.PaperTiger:ActionCollection.Fieldnames) < prototype(Neos.Fusion:Component) {
+    nodePath = null
+    renderer = afx`
+        <div>
+            Fieldnames:&nbsp;
+            <Neos.Fusion:Loop @glue=", " items={q(node).children(props.nodePath).find('[instanceof Sitegeist.PaperTiger:Field]').get()} itemName="field">
+                {'{' + q(field).property("name") + '}'}
+            </Neos.Fusion:Loop>
+        </div>
+    `
```

Comment:
I would like to have this value configurable, and I event would switch to default Fusion, to make it easier to change the look and view (Add CSS class or change the tag name for example)
```suggestion
    describtion = 'Fieldnames:&nbsp;'
    renderer = Neos.Fusion:Tag {
        content = Neos.Fusion:Join {
            describtion = ${props.describtion}
            fieldNames = Neos.Fusion:Loop {
                @glue = ', '
                items = ${q(node).children(props.nodePath).find('[instanceof Sitegeist.PaperTiger:Field]').get()}
                itemName = 'field'
                itemRenderer = ${'{' + q(field).property('name') + '}'}
            }
        }
    }
```